indy_node/utils/node_control_utils.py

```diff
@@ -30,8 +30,7 @@ class NodeControlUtil:
     @classmethod
     def run_shell_command(cls, command, timeout=TIMEOUT):
```

Comment:
Please add comments about the difference and usage for `run_shell_command` and `run_shell_script`.

Reply:
done